examples/README.md

```diff
@@ -0,0 +1,61 @@
+| CHANNEL_MONITOR | OT_CHANNEL_MONITOR | 0 | Enables support for channel monitor. | Enable this switch on devices that are supposed to determine the cleaner channels. |
+| COAP | OT_COAP | 0 | Enables support for the CoAP API. | Enable this switch if you want to control Constrained Application Protocol communication. |
+| COAPS | OT_COAPS | 0 | Enables support for the secure CoAP API. | Enable this switch if you want to control Constrained Application Protocol Secure (CoAP over DTLS) communication. |
+| COAP_OBSERVE | OT_COAP_OBSERVE | 0 | Enables support for CoAP Observe (RFC7641) API. |  |
+| COMMISSIONER | OT_COMMISSIONER | 0 | Enables support for Commissioner. | Enable this switch on device that is able to perform Commissioner role. CMake swtich: . |
+| COVERAGE | OT_COVERAGE | 0 | Enables the generation of code-coverage instances. |  |
+| CHILD_SUPERVISION | OT_CHILD_SUPERVISION | 0 | Enables support for child supervision. | Enable this switch on a parent or child node with custom OpenThread application that manages the supervision, checks timeout intervals, and verifies connectivity between parent and child. Read more about the child supervision: https://openthread.io/guides/build/features/child-supervision. |
+| CLI_TRANSPORT | not implemented | UART | Selects the transport of CLI. | You can set this switch to UART (default) or CONSOLE. |
+| CSL_RECEIVER | OT_CSL_RECEIVER | 0 | Enables CSL receiver feature for Thread 1.2. |  |
+| DEBUG | not implemented | 0 | Allows building debug instance. Code optimization is disabled. |  |
+| DHCP6_CLIENT | OT_DHCP6_CLIENT | 0 | Enables support for the DHCP6 client. | The device is able to act as typical DHCP client. Enable this switch on a device that is supposed to request networking parameters from the DHCP server. |
+| DHCP6_SERVER | OT_DHCP6_SERVER | 0 | Enables support for the DHCP6 server. | The device is able to act as typical DHCP server. Enable this switch on a device that is supposed to provide networking parameters to devices with DHCP_CLIENT switch enabled. |
+| DIAGNOSTIC | OT_DIAGNOSTIC | 0 | Enables diagnostic support. | Enable this switch on a device that is tested in the factory production stage. |
+| DISABLE_DOC | not implemented | 0 | Disables building of the documentation. |  |
+| DISABLE_TOOLS | not implemented | 0 | Disables building of tools. |  |
+| DNS_CLIENT | OT_DNS_CLIENT | 0 | Enables support for DNS client. | Enable this switch on a device that sends a DNS query for AAAA (IPv6) record. |
+| DUA | OT_DUA | 0 | Enables the Domain Unicast Address feature for Thread 1.2. |  |
+| DYNAMIC_LOG_LEVEL | not implemented | 0 | Enables the dynamic log level feature. | Enable this switch if OpenThread log level is required to be set at runtime. Read more about logging: https://openthread.io/reference/group/api-logging. |
```

Comment:
Would it make more sense to link to the Logging guide rather than the reference?  For this and other related switches like LOG_OUTPUT, REFERENCE_DEVICE, FULL_LOGS etc.

https://openthread.io/guides/build/logs